<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin_Check</class>
 <widget class="QDialog" name="Admin_Check">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>839</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tableView_display">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>651</width>
     <height>391</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_refresh">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>20</y>
     <width>151</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>刷新列表</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_back">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>380</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_shipment">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>80</y>
     <width>141</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>显示出货单</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Purchase">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>110</y>
     <width>141</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>显示进货单</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_OnlyMe">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>170</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>不显示他人处理的</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_OnlySuspend">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>140</y>
     <width>141</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>只显示缺货挂起</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
